modules/libarchive/3.7.3/presubmit.yml

```diff
@@ -0,0 +1,27 @@
+bcr_test_module:
+  module_path: ""
+  matrix:
+    platform:
+      - debian10
+      - debian11
+      - ubuntu2004
+      # - ubuntu2004_arm64
```

Comment:
how did you generate the patch? I've seen patches generated with `git diff` work but `git show` not work in the past. Maybe it would help to trim the header lines?

Reply:
I just used `git diff`. I have no idea why its not working. This is how I'm currently generating it https://github.com/zaucy/libarchive/blob/7cb94b082c11bd171ac219fbb89f9517f7f23526/bazel/update_bcr.nu#L25

Reply:
hmm, maybe @Wyverald has an idea?